packages/malloy-db-test/src/postgres.spec.ts

```diff
@@ -20,6 +20,19 @@ if (runtime === undefined) {
   throw new Error("Couldn't build runtime");
 }
 
+// Idempotently create schema and tables with capital letters to use in tests.
+beforeAll(async () => {
+  await runtime.connection.runSQL('create schema if not exists "UpperSchema";');
+  await Promise.all([
+    runtime.connection.runSQL(
+      'create table if not exists "UpperSchema"."UpperSchemaUpperTable" as select 1 as one;'
+    ),
+    runtime.connection.runSQL(
+      'create table if not exists "UpperTablePublic" as select 1 as one;'
+    ),
+  ]);
+});
```

Comment:
these could instead be part of `test/sql/malloytest-postgres.sql.gz` that we were just discussing, though I someone would need to run it one time on your side.